plugins/jsconnect/views/settings.php

```diff
@@ -14,7 +14,29 @@
 </div>
 <div class="FilterMenu"><?php
     echo Anchor(T('Add Connection'), '/settings/jsconnect/addedit', 'SmallButton');
-    ?></div>
+?></div>
+<div class="Info">
+    <h2>Signing In</h2>
+    <?php
+        echo $this->Form->open();
+        echo $this->Form->errors();
+
+        echo wrap($this->Form->checkBox(
+            'Garden.Registration.AutoConnect',
+            'Automatically connect users based on e-mail address.'
```

Comment:
'Automatically connect to an existing user account if it has the same email address.'